.github/workflows/Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM centos as centos-python
```

Comment:
This file should really be named `Dockerfile-centos`.  (Out of scope for this PR.)